axes/attempts.py

```diff
@@ -33,29 +33,32 @@ def filter_user_attempts(request, credentials: dict = None) -> QuerySet:
 
     username = get_client_username(request, credentials)
 
-    filter_kwargs = get_client_parameters(
+    filter_kwargs_list = get_client_parameters(
         username, request.axes_ip_address, request.axes_user_agent
     )
-
-    return AccessAttempt.objects.filter(**filter_kwargs)
```

Comment:
The reason, why I couldn't do this is one QuerySet is, that we need to return list of AccessAttempt sets. That is because we sum up number of failed attempts in each AccessAttempt set individually. Otherwise the count of failed attempts would be 3 instead of 2 in this set of attempts:
```
IP1 U1
IP2 U1
IP2 U2
```

Reply:
BTW this doesn't increase requirements much - in current implementation the max number of QuerySets in the list is 2.